src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/InterventionDTO.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Intervention
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class InterventionDTO(
+  val id: UUID? = null,
+  val name: String? = null,
+  val createdAt: OffsetDateTime? = null
```

Comment:
i wonder if `lastUpdated` might be a more useful field here instead of `createdAt`?

Reply:
Not sure tbh - will we ever let the intervention be updated?  if so would we want both? For auditing, do we need to link through to createdBy and updatedBy  through to the new auth_user table?  Or are we not bothered at this stage?